viewer/src/main/java/org/kaazing/monitoring/agrona/viewer/task/MetricsTask.java

```diff
@@ -0,0 +1,44 @@
+package org.kaazing.monitoring.agrona.viewer.task;
+
+import java.util.List;
+import java.util.concurrent.ScheduledFuture;
+
+import org.kaazing.monitoring.reader.api.Metric;
+import org.kaazing.monitoring.reader.api.MetricsCollector;
+
+/**
+ * Interface for tasks responsible with retrieving collector messages and metrics
+ *
+ */
+public interface MetricsTask {
```

Comment:
Three methods in this interface are never used (except by tests), I would suggest removing them (they are getScheduledTask, getMetricsList, and getMetricsCollector)